icons/cctv.svg

```diff
@@ -0,0 +1,16 @@
+<svg
```

Comment:
To me, the icon is a bit too detailed, especially the camera lens part. If you compare it with like the `camera` or `video` icons. I think should be nice if they match.

Here some inspiration of CCTV icons that are more simplified 

![image](https://github.com/lucide-icons/lucide/assets/11825403/8078882a-597e-41d1-8b74-b629998cd0b4)
![image](https://github.com/lucide-icons/lucide/assets/11825403/5f59dd19-c7e8-467e-9774-63a8cd6cc5fe)

Reply:
@ericfennis See latest `push`, and/or these alts:

![preview](https://github.com/lucide-icons/lucide/assets/7797479/c0cbf84e-bee8-47e8-ab37-4056a53e430b)

cc: @karsa-mistmere @jguddas

Reply:
I'm for the last one. It's simple and friendly. The other ones looks a bit "evil" in don't know 😅 
And it matches a bit with the other video icon

Reply:
> looks a bit "evil"

I think CCTV cameras are supposed to seem a bit sinister 😅

I really like the first one best. But I suppose it comes down to another vote (need something like this [bot](https://github.com/marketplace/actions/github-polls-bot-action) again)…